README.md

```diff
@@ -28,10 +28,10 @@ $ docker run -it --net=host --rm timescale/timescaledb psql -h localhost -U post
 
 You can also connect your app via port `5432` on the host machine.
 
-Note that you can also set an environmental variable, `TIMESCALEDB_TELEMETRY`, to set the level of [telemetry](https://docs.timescale.com/using-timescaledb/telemetry) in the Timescale docker instance. For example, to turn off telemetry, run:
+If you are running your docker image for the first time, you can also set an environmental variable, `TIMESCALEDB_TELEMETRY`, to set the level of [telemetry](https://docs.timescale.com/using-timescaledb/telemetry) in the Timescale docker instance. For example, to turn off telemetry, run:
 
 ```
 $ docker run -d --name some-timescaledb -p 5432:5432 --env TIMESCALEDB_TELEMETRY=off timescale/timescaledb
 ```
 
-The default telemetry level is `basic`.
+Note that if the docker image has previously been initialized, you should not use this environment variable to set the level of telemetry. Instead, follow the [instructions](https://docs.timescale.com/using-timescaledb/telemetry) in our docs to disable telemetry once a cluster is running.
```

Comment:
Minor nit about wording: if the postgres cluster has previously been initialized not necessarily the docker image